app/controllers/admin/words_controller.rb

```diff
@@ -0,0 +1,60 @@
+class Admin::WordsController < AdminController
+  def new
+    @category = Category.find(params[:category_id])
+    @word = @category.words.build # 特定のカテゴリIDをもつ空のレコードを作る
+    3.times {
+      @word.choices.build
+    } 
+  end
+
+  def index
+    @category = Category.find(params[:category_id])
+    @words = Word.where(category_id: params[:category_id]) # /:id/に一致するwordのみ表示する
```

Comment:
You can refactor this code and say `@words = @category.words` <- this will give you save results with less code